lib/inspec-iggy/version.rb

```diff
@@ -4,8 +4,8 @@
 #
 # Copyright:: 2018, Chef Software, Inc <legal@chef.io>
 #
-
-# Iggy version
-module Iggy
-  VERSION = '0.3.0'.freeze
+module InspecPlugins
+  module Iggy
+    VERSION = '0.3.0'.freeze
```

Comment:
This can go to 0.4.0

Reply:
We'll make sure it gets bumped after merge to master.